config/routes.yaml

```diff
@@ -1,11 +1,6 @@
 # This file is the entry point to configure your own HTTP routes.
 # Files in the routes/ subdirectory configure the routes for your dependencies.
 
-# ===== Api =====
```

Comment:
I think this is an unintentional change. please revert!

Reply:
thx